tests/integration/tests.mk

```diff
@@ -12,14 +12,6 @@ ifneq ($(CI),)
 	_INTEGRATION_TEST_FLAGS += --istio.test.pullpolicy=IfNotPresent
 endif
 
-ifeq ($(TEST_ENV),minikube)
```

Comment:
how does this work? We always install metallb even for single cluster?

Reply:
I guess for TOPOLOGY != Multicluster we could pass the flag to say there isn't loadbalancer support. I actually want to drop this in favor of something where we check to see if Kube will support externalip on loadbalancer.

Reply:
Brought this back, setting TEST_ENV=kind-metallb for multicluster.